Lab1/Geometry/Shapes.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace Geometry.ShapeList
+{
+    public sealed class Shapes : IList<Shape>
```

Comment:
this is already an overkill because you don't have any new logic to override ICollection(IList in your case). But good job 👍